lib/Doctrine/DBAL/Types/DateIntervalType.php

```diff
@@ -22,7 +22,7 @@ public function getName()
      */
     public function getSQLDeclaration(array $fieldDeclaration, AbstractPlatform $platform)
     {
-        $fieldDeclaration['length'] = 20;
+        $fieldDeclaration['length'] = 255;
```

Comment:
Tweaking is not really up to us - we just need to accommodate a generic use-case scenario, and that's it.